docs/changelog/changelog_en.md

```diff
@@ -1,44 +1,41 @@
 # Changelog PlentyONE Shop
 
-## v1.x.x (2025-xx-xx)
+## v1.9.0 (2025-01-23) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.8.0...v1.9.0" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
 
 ### New
 
-- Added Nuxt security module.
-- Added page for shipping legal text.
-- Added the logic to render components within a module in certain areas of the PWA
-- Added delivery days to checkout shipping providers.
-- Added robots for category page.
-- Subtitle, pretitle and title options where added to the Recommened Products component
-- Added canonical for item page.
-- Media Card is now Image Text
-- The newsletter works as the other available blocks. (using the JSON as the source of the data)
+- The [Nuxt security module](https://nuxt-security.vercel.app/) has been added to the web app.
+- The web app is now equipped to [render components from a module](https://pwa-docs.plentymarkets.com/guide/how-to/module/inject-components) in designated areas of the shop.
+- The shop now has a dedicated page for legal information regarding shipping.
```

Comment:
Isn't it more like users can configure a category from which to take the markup for the shipping information page, which is then automatically linked? Also what's the default URL of that, if it's configured? @abocsan-plenty

Reply:
```suggestion
- The shop now has a dedicated legal information page for shipping information. You can link a category inside the Online Store Section of the Shops Configuration. The template configured on that category is used for the pages content. The URL for the page is /shipping.
```